Changes

```diff
@@ -138,7 +138,7 @@ Working version
 
 ### Internal/compiler-libs changes:
 
-- GPR#1488: Refreshing parmatch
+- GPR#1488, GPR#1561: Refreshing parmatch
```

Comment:
I wouldn't add that here.
I would add it to your other GPR which started changing the names. (so yes, I would wait for the other to be merged and then rebase this PR, if I were you)